UPGRADE-1.11.md

```diff
@@ -20,7 +20,7 @@ This obvious dev dependency was part of Sylius requirements. In 1.11 we've moved
 
 Due to the drop of support PHP `7.4` Sylius also will not support it since version `1.11`.
 
-### Potential BC-break
+### Potential BC-break: state machine callbacks
```

Comment:
Can you sub-sections to `Potential BC-break` instead, like this:

```markdown
### Potential BC-break

#### State machine callbacks

#### Form type extensions
```

Similar to https://github.com/Sylius/Sylius/pull/13531/files.

Reply:
Done